test/test_common/network_utility.cc

```diff
@@ -105,9 +105,7 @@ const std::string addressVersionAsString(const Address::IpVersion version) {
 Address::InstanceConstSharedPtr getSomeLoopbackAddress(Address::IpVersion version) {
   if (version == Address::IpVersion::v4) {
     // Pick a random address in 127.0.0.0/8.
-    // TODO(jamessynge): Consider how to use $GTEST_RANDOM_SEED for seeding the rng.
-    // Perhaps we need a TestRuntime::getRandomGenerator() or similar.
-    Runtime::RandomGeneratorImpl rng;
+    TestRandomGenerator rng;
```

Comment:
If we're going to have multiple RNGs per test (which this might imply), maybe we want to set a global seed in a static initializor for the class or something like that. Otherwise, if we have a test failure, it's unclear which seed to use to replay if there are multiple.